form.spec

```diff
@@ -0,0 +1,34 @@
+### RPM external form 4.1.033e
+Source: https://gosam.hepforge.org/gosam-installer/form-%{realversion}.tar.gz
+
+
+%if "%{?cms_cxx:set}" != "set"
+%define cms_cxx g++
+%endif
+
+
+%prep
+%setup -q -n form-4.1
+
+
+%build
+
+
+
+CXX="$(which %{cms_cxx}) -fPIC"
+CC="$(which gcc) -fPIC"
+PLATF_CONF_OPTS="--enable-shared --disable-static"
+
+./configure $PLATF_CONF_OPTS \
+            --prefix=%i \
+            --bindir=%i/bin \
+            --without-gmp \
```

Comment:
I first tried it with gmp but always the libgmp.so.3 was not, thus, I removed it in the form.spec and forgot to do it in the form-toolfile.spec, too.